### PR TITLE
#714 added rename feature to TkStart

### PR DESCRIPTION
**Pull request by dmzaytsev**
For #714

* added rename feature to TkStart
* added a unit test

### CODE REVIEW

**Comment by karato** [comment]
@dmzaytsev Thanks for your pull request, let me find someone who can review it

**Comment by karato** [comment]
@pinaf review it

File: netbout-web/src/main/java/com/netbout/rest/TkStart.java [hunk 1]
```diff
@@ -104,6 +95,50 @@ public Response act(final Request req) throws IOException {
...
+        final Iterator<String> invite = href.param("invite").iterator();
+        while (invite.hasNext()) {
+            final String friend = invite.next();
+            try {
+                bout.friends().invite(friend);
+            } catch (final Friends.UnknownAliasException ex) {
+                this.discard(bout);
+                throw new RsFailure(ex);
+            }
+            msg.append(String.format(", \"%s\" invited", friend));
```
Comment (pinaf) [inline netbout-web/src/main/java/com/netbout/rest/TkStart.java]: @dmzaytsev this message should only be appended if the invite succeeded, don't you think?

**Comment by pinaf** [comment]
@dmzaytsev also, in theory you could rename multiple times at once since you are looping through the iterator. Maybe we should limit that to at most 1 rename?

File: netbout-web/src/main/java/com/netbout/rest/TkStart.java (hunk 1 above)
Comment (dmzaytsev) [inline netbout-web/src/main/java/com/netbout/rest/TkStart.java]: @pinaf you're right. will `sent the invitation to %s` be good?

**Comment by dmzaytsev** [comment]
@pinaf done
please take a look

File: netbout-web/src/main/java/com/netbout/rest/TkStart.java (hunk 1 above)
Comment (pinaf) [inline netbout-web/src/main/java/com/netbout/rest/TkStart.java]: @dmzaytsev the text is fine. I was refering to the location. Perhaps move it to line 111?

**Comment by pinaf** [comment]
@dmzaytsev thanks. 1 comment above.

**Comment by dmzaytsev** [comment]
@pinaf I misunderstood you 
sure, you're absolutely right
done, please take a look

**Comment by pinaf** [comment]
@rultor merge

**Comment by pinaf** [comment]
@rultor deploy

**Comment by dmzaytsev** [comment]
@rultor deploy

**Comment by dmzaytsev** [comment]
@pinaf thanks for the review )

**Comment by karato** [comment]
@ypshenychka please, review this task for compliance with our [quality rules](http://www.teamed.io/qa.html)

**Comment by karato** [comment]
@rultor deploy now pls

**Comment by ypshenychka** [comment]
@pinaf 
According to our [QA Rules](http://www.teamed.io/qa.html):
>The code reviewer found at least three problems in the code.

Two issues were addressed during code review. 
Please confirm that you'll try to find at least three issues next time.

**Comment by pinaf** [comment]
@ypshenychka yes, I confirm

**Comment by ypshenychka** [comment]
@karato Quality is acceptable.

**Comment by karato** [comment]
> @karato Quality is acceptable.

@ypshenychka thank you, let's try to make it "good" next time

**Comment by karato** [comment]
@pinaf 10 mins was added to the account of @ypshenychka (for QA review), in transaction 64002802; Thanks for your contribution, **20 mins** was added to your account, payment ID is `AP-8G732095PP1622826`, 48 hours and 39 mins spent total; extra minutes for review comments (c=5); +20 added to your rating, current score is: [+4627](http://www.netbout.com/b/35128?open=rating)

**Pull request merged**